src/almanack/metrics/metrics.yml

```diff
@@ -3,66 +3,56 @@ metrics:
   - name: "repo-path"
     id: "SGA-META-0001"
     result-type: "str"
-    result-data-key: "repo_path"
```

Comment:
I am a fan of subtraction to add clarity!

Reply:
Thanks!